lms/static/js/spec/learner_dashboard/program_details_header_spec.js

```diff
@@ -0,0 +1,59 @@
+define([
+        'backbone',
+        'jquery',
+        'js/learner_dashboard/views/program_header_view'
+    ], function (Backbone, $, SidebarView) {
+        
+        'use strict';
+        
+        describe('Program Details Header View', function () {
+            var view = null,
+                programModel,
+                context = {
+                    uuid: '12-ab',
+                    name: 'Astrophysics',
+                    subtitle: 'Learn contemporary astrophysics from the leaders in the field.',
+                    category: 'xseries',
+                    organizations: [
+                        {
+                            display_name: 'Australian National University',
+                            img: 'https://edxuploads.s3.amazonaws.com/organization_logos/logo-anux.png',
+                            key: 'ANUx'
+                        }
+                    ],
+                    banner_image_urls: {
+                        w1440h480: 'http://placehold.it/1440x480',
```

Comment:
Having unit tests depend on external systems makes me nervous. Is there a local alternative?